.gitignore

```diff
@@ -21,6 +21,7 @@ tags
 tester.py
 tags
 .idea/
+.run/
```

Comment:
Can this be removed?

Reply:
I can remove it, but this is the new location that JetBrains IDE favor to store configurations instead of `.idea/`, so they might be introduced by someone unintentionally.